.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,25 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Coverage Report
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set JDK ${{ matrix.java-version }}
```

Comment:
@GracianoR I could be wrong, but I expected a variable definition like this for this to work. 
`matrix:`    
        `os: [ubuntu-latest]`  
        `java-version: [8]`   
See example [here](https://github.com/eclipse/eclipse-collections/blob/master/.github/workflows/acceptance.yml) - line 12-14